### ISSUE TITLE
error in log

### ISSUE THREAD

**Issue by M0UL** [zone117x/Jackett#30]
i found this error in the log. i am using jacket version 2.6

2015-05-05 11:26:05.5563 Error Unexpected character encountered while parsing value: <. Path '', line 0, position 0.Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Jackett.Indexers.MoreThanTV.<PerformQuery>d__e.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Indexers\MoreThanTV.cs:line 200
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Jackett.Server.<ProcessTorznab>d__10.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Server.cs:line 196
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Jackett.Server.<ProcessHttpRequest>d__8.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Server.cs:line 135 
 Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Jackett.Indexers.MoreThanTV.<PerformQuery>d__e.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Indexers\MoreThanTV.cs:line 200
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Jackett.Server.<ProcessTorznab>d__10.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Server.cs:line 196
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Jackett.Server.<ProcessHttpRequest>d__8.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Server.cs:line 135

2015-05-05 11:26:05.8336 Debug Found 2 releases from Strike

**Comment by zone117x** [comment]
Look for a morethantv or iptorrents error file next to the log and paste them. I need to see the results page for whatever you were searching/browsing that is causing the error.

**Comment by M0UL** [comment]
i found these error files Error on 20150505112605 for MoreThanTV.txt;
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.ReadInternal()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader)
   at Newtonsoft.Json.Linq.JObject.Parse(String json)
   at Jackett.Indexers.MoreThanTV.<PerformQuery>d__e.MoveNext() in c:\Users\Matthew\Source\Repos\Jackett\src\Jackett\Indexers\MoreThanTV.cs:line 166




<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Login :: morethan.tv</title>
<meta http-equiv="X-UA-Compatible" content="chrome=1; IE=edge" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="favicon.ico" />
		<link href="static/login_v2/css/main.css" rel="stylesheet" type="text/css" />
		<script src="static/functions/jquery.js" type="text/javascript"></script>
        <script src="static/functions/script_start.js?v=1409393233" type="text/javascript"></script>
        <script src="static/functions/ajax.class.js?v=1409393233" type="text/javascript"></script>
        <script src="static/functions/cookie.class.js?v=1409393233" type="text/javascript"></script>
        <script src="static/functions/storage.class.js?v=1409393233" type="text/javascript"></script>
        <script src="static/functions/global.js?v=1409393233" type="text/javascript"></script>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-2" />
<meta name="keywords" content="">
<meta name="description" content="">
<meta name="abstract" content="">
<meta name="author" content="dracu.tv">
<meta name="robots" content="INDEX, FOLLOW">
<meta name="revisit-after" content="3 days" />
<meta name="expires" content="never"/>
</head>

<body>
        
        <div class="top">
                <div class="center">
                        <a class="logo" href="/"><img src="/static/login_v2/img/logo.png" width="566" height="231" alt="" /></a>
                </div>
        </div>
        

        
        <div class="mid">
                <div class="center">

                        <div class="menu">
                                <ul>
                                        <li><a  href="index.php">Home</a></li>
                                        <li><a  class="active"  href="login.php">Login</a>
                                        <li><a  href="register.php">Register</a></li>
                                </ul>
                        </div>

                        <div class="clear"></div>

                        <div class="content">
<span id="no-cookies" class="hidden warning">You appear to have cookies	disabled.<br />
<br />
</span>
<noscript>
	<span class="warning">morethan.tv requires JavaScript to function properly. Please enable JavaScript in your browser.</span><br />
	<br />
</noscript>
<form class="auth_form" name="login" id="loginform" method="post"
	action="login.php">
	<table class="layout" border="0">
		<tr>
			<td colspan="2"><input class="form_user" type="text" name="username"
				id="username" class="inputtext" required="required" maxlength="20"
				pattern="[A-Za-z0-9_?]{1,20}" autofocus="autofocus"
				placeholder="Username" /></td>
		</tr>
		<tr>
			<td colspan="2"><input class="form_pass" type="password"
				name="password" id="password" class="inputtext" required="required"
				maxlength="100" pattern=".{6,100}" placeholder="Password" /></td>
		</tr>
		<tr><td colspan="2">&nbsp</td></tr>
		<tr>
			<td><input type="submit" name="login" value="Log in" class="submit" /></td>
			<div class="clear"></div>
		</tr>
		<div class="clear"></div>

		<table class="layout" border="0">
			<tr>
				<td class="forgotp">
								</td>
				<td class="checkbox"><input type="checkbox" id="keeplogged"	name="keeplogged" value="1"
					 />
					<label for="keeplogged">Remember me</label></td>
			</tr>
	</table>
</form>
<script type="text/javascript" src="static/functions/detect_mobile.js"></script>
<script type="text/javascript">
cookie.set('cookie_test', 1, 1);
if (cookie.get('cookie_test') != null) {
	cookie.del('cookie_test');
} else {
	$('#no-cookies').gshow();
}
</script>
                        </div>
                </div>
        </div>
        

        
        <div class="bot">
        </div>
        
</body>
</html>

**Comment by zone117x** [comment]
Thanks for the error logs @M0UL, the parsing bug with morethantv is fixed in latest code. Will close issue when I get the next release build out.

**Comment by zone117x** [comment]
These should be fixed now.
